CameraColorPicker/app/src/kids/res/layout/view_palette_list_page.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<merge xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <FrameLayout
+        android:id="@+id/view_palette_list_empty_view"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
```

Comment:
sure I totally forgot to remove that. Initially, a additionnal arrow was also contained by this FrameLayout. Gonna removed it right now.